### ISSUE TITLE
Dockerfile FFMPEG missing

### ISSUE THREAD

**Issue by DunklerPhoenix** [wger-project/wger#1016]
In the dockerfile is ffmpeg missing and should be added for video functionality.

**Comment by rolandgeider** [comment]
Hi! Adding ffmepg increases the image size by something around 400mb if I remember correctly and since this is pretty optional, I decided to leave it for the time being. Uploading should work without ffmepg, but there won't be any  validation.

**Comment by DunklerPhoenix** [comment]
okay. You could either create a second image with ffmpeg or you can add an option to the entrypoint script that will add ffmpeg (and additional packages which are passed by env. The second solution is for example used by photoprism. There you have a field in the config file where you can add additional packages.

**Comment by rolandgeider** [comment]
Yes, we could do something like that. I'll take a look at photoprism

**Comment by DunklerPhoenix** [comment]
You can set the Env (e.g.) PHOTOPRISM_INIT: "gpu, tensorflow, nano"

Here is the entry for it https://github.com/photoprism/photoprism/blob/cb89a1a189aede610d4f8c7455232d299c1a4dca/scripts/dist/entrypoint-init.sh#L68

and there are the init scripts:
https://github.com/photoprism/photoprism/tree/cb89a1a189aede610d4f8c7455232d299c1a4dca/scripts/dist
